arviz/plots/__init__.py

```diff
@@ -9,7 +9,7 @@
 from .forestplot import plot_forest
 from .hpdplot import plot_hpd
 from .jointplot import plot_joint
-from .kdeplot import plot_kde, _fast_kde, _fast_kde_2d
+from .kdeplot import plot_kde, _fast_kde_2d
```

Comment:
You can also move `_fast_kde_2d` next to `_fast_kde`, sorry for not saying it explicitly before